task/05-regex-tasks.js

```diff
@@ -31,7 +31,7 @@
  * @return {RegExp}
  */
 function getRegexForGuid() {
-   throw new Error('Not implemented');
+   return /{[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{12}}/;
 }
```

Comment:
three times repeating part [0-9A-Fa-f]{4}- could be reduced to ([0-9A-Fa-f]{4}-){3}